src/together/resources/finetune.py

```diff
@@ -366,17 +380,78 @@ def list_events(self, id: str) -> FinetuneListEvents:
             ),
             stream=False,
         )
-
         assert isinstance(response, TogetherResponse)
 
         return FinetuneListEvents(**response.data)
 
+    def list_checkpoints(self, id: str) -> List[FinetuneCheckpoint]:
+        """
+        List available checkpoints for a fine-tuning job
+
+        Args:
+            id (str): Unique identifier of the fine-tune job to list checkpoints for
+
+        Returns:
+            List[FinetuneCheckpoint]: List of available checkpoints
+        """
+        events = self.list_events(id).data or []
+
+        checkpoints: List[FinetuneCheckpoint] = []
+
+        for event in events:
+            event_type = event.type
+
+            if event_type == FinetuneEventType.CHECKPOINT_SAVE:
+                formatted_time = format_event_timestamp(event)
+                step = get_event_step(event)
+                checkpoint_name = f"{id}:{step}" if step else id
```

Comment:
step=0 will put the code in a surprise :)
```suggestion
                checkpoint_name = f"{id}:{step}" if step is not None else id
```